lib/migration/plugins/init.go

```diff
@@ -20,5 +20,5 @@ import (
 	_ "github.com/vmware/vic/lib/migration/plugins/plugin1"
 	_ "github.com/vmware/vic/lib/migration/plugins/plugin2"
 	_ "github.com/vmware/vic/lib/migration/plugins/plugin5"
-	_ "github.com/vmware/vic/lib/migration/plugins/plugin7"
+	_ "github.com/vmware/vic/lib/migration/plugins/plugin6"
```

Comment:
Did you try this?
`You can try to update this line https://github.com/vmware/vic/blob/master/lib/migration/migrator_test.go#L35 to -1, because you have a plugin registered with latest version.`